tools/test-zulip.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
```

Comment:
I first had this script named `test-zulip-api.py`, but `test-zulip` feels better when considering the names `test-bots` and `test-botserver` (all names match their corresponding package names)